neo4j/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# integration pip requirements
+time
+requests
+hashlib
```

Comment:
hashlib is part of the standard library so does not need to be included as a dependency.